.zenodo.json

```diff
@@ -1,5 +1,5 @@
 {
-    "description": "pure-Python implementation of HistFactory models with tensors and autograd",
+    "description": "pure-Python implementation of HistFactory models with tensors and autodiff",
```

Comment:
We should probably also decide on one description and just use that across `setup.cfg`, `.zenodo.json`, and `codemeta.json`.